packages/orchestration/src/facade.js

```diff
@@ -78,6 +78,13 @@ export const makeOrchestrationFacade = ({
     const [wrappedCtx] = prepareEndowment(subZone, 'endowments', [hostCtx]);
     const hostFn = asyncFlow(subZone, 'asyncFlow', guestFn);
 
+    deepMapObject(
```

Comment:
nb: the usage here somewhat abuses the "map" idea. Kind of a `deepAssert` or `visitObject`

Reply:
quite